quiche/src/lib.rs

```diff
@@ -2247,6 +2247,150 @@ impl Connection {
         self.encode_transport_params()
     }
 
+    /// Sets the congestion control algorithm used.
```

Comment:
I would consider a `set_recovery_config_in_handshake` method that takes a `RecoveryConfig` as a way to minimize future plumbing if additional fields are added to `RecoveryConfig`.  But this works.

Reply:
The problem is that not all fields in RecoveryConfig can be changed with this method, e.g. `max_ack_delay` is negotiated via transport parameters. And even if that was supported then we'd need to make sure any future config option is also compatible, which seems kind of limiting.